genTriposGAFFandGAFF2.py

```diff
@@ -15,11 +15,28 @@
 import openeye.oechem as oechem
 
 def make_path(filename):
+    """
+    This function makes subdirectories.
+    """
     path = os.path.split(filename)[0]
     if not os.path.exists(filename):
         os.makedirs(path)
 
 def GenTriposGAFF(mol):
+    """
+    This function takes a molecule from a given SDF file
+    and charges it to make a tripos mol2. Tripos mol2 are
+    used to generate GAFF and GAFF2 mol2, inpcrd and prmtop 
+    files.
```

Comment:
Let's be a bit more precise with the description. The SDF file itself is not charged. Also, technically, we could read in molecules from *.mol2 files, so we don't need to mention the SDF part in this function specifically.
You could say something along the lines of,
> This function reads in an OEChem molecule, charges it using the AM1-BCC scheme, and generates a Tripos mol2 file. The Tripos mol2 file is used to generate GAFF and GAFF2 \*.mol2 files, topology files (\*.prmtop), and coordinate files (\*.inpcrd).